content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -14,35 +14,35 @@
   </head>
   <body>
     <main>
-      <h1>Landmark Regions</h1>
+      <h1>ランドマーク領域</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="introduction">
-        <h2>Introduction</h2>
+        <h2>はじめに</h2>
         <p>
-          ARIA landmark roles provide a powerful way to identify the organization and structure of a web page.
-          By classifying and labelling sections of a page, they enable structural information that is conveyed visually through layout to be represented programmatically.
-          Screen readers exploit landmark roles to provide keyboard navigation to important sections of a page.
-          Landmark regions can also be used as targets for &quot;skip links&quot; and by browser extensions to enhanced keyboard navigation.
+          ARIAランドマークロールは、ウェブページの組織と構造を識別する強力な方法を提供します。
+          ページのセクションを分類し、ラベルを付けることで、レイアウトを通じて視覚的に伝えられる構造情報をプログラムで表現できるようにします。
+          スクリーンリーダーは、ページの重要なセクションへのキーボードナビゲーションを提供するために、ランドマークロールを活用します。
+          ランドマーク領域は、&quot;スキップリンク&quot;のターゲットとして、また、キーボードナビゲーションの強化にブラウザ拡張機能で使用することもできます。
         </p>
-        <p>This section explains how HTML sectioning elements and ARIA landmark roles are used to make it easy for assistive technology users to understand the meaning of the layout of a page.</p>
+        <p>このセクションでは、HTMLセクション要素とARIAランドマークロールが、支援技術の利用者がページのレイアウトの意味を容易に理解するためにどのように使用されるかを説明します。</p>
       </section>
 
       <section>
-        <h2>HTML Sectioning Elements</h2>
+        <h2>HTMLセクショニング要素</h2>
         <p>
-          Several HTML sectioning elements automatically create ARIA landmark regions.
-          So, in order to provide assistive technology users with a logical view of a page, it is important to understand the effects of using HTML sectioning elements.
-          [[HTML-ARIA]] contains more information on HTML element role mapping.
+          いくつかのHTMLセクショニング要素は、自動的にARIAランドマーク領域を作成します。
+          したがって、支援技術の利用者にページの論理的なビューを提供するためには、HTMLセクショニング要素を使用する際の効果を理解することが重要です。
```

Comment:
soをしたがってと取るには固すぎるかな…と。

```suggestion
          よって、支援技術の利用者にページの論理的なビューを提供するためには、HTMLセクショニング要素を使用する際の効果を理解することが重要です。
```